recipe/recipe.yaml

```
@@ -11,7 +11,7 @@ source:
   sha256: 1ba857408aff59ce5c1dfab052b2e94ab9cca14253778b8f9da15208887b16f6
 
 build:
-  number: 0
+  number: 1
```

Note that we can keep this build number at 0 since the first build hasn't been uploaded yet. Wait for https://github.com/conda-forge/admin-requests/pull/1504

```suggestion
  number: 0
```